examples/README.md

```diff
@@ -37,6 +37,14 @@ APIs:
 API Examples
 ------------
 
+### [Asset](asset)
+
+- [Utilization](asset/utilization): Demonstrates how to use the SystemLink Asset Management
+  Client API to track asset utilization.
+
+  Run the example on a machine with SystemLink Client. If the system is not managed, utilization data is stored locally on the system.
```

Comment:
With my previous comment, I intended to say you should create an additional README.md file in the examples/asset directory that contains this information. For example: https://github.com/ni/systemlink-client-docs/tree/a72c9c56247f103939e65b2c968f562add5403b5/examples/message